processor/README.md

```diff
@@ -2,3 +2,69 @@
 *Note* This documentation is still in progress. For any questions, please reach
 out in the [OpenTelemetry Gitter](https://gitter.im/open-telemetry/opentelemetry-service)
 or refer to the [issues page](https://github.com/open-telemetry/opentelemetry-service/issues).
+
+## <a name="attributes"></a>Attributes Processor
+The attributes processor modifies attributes of a span.
+
+It takes a list of actions which are performed in order specified in the config.
+The supported actions are:
+- inserts: Inserts a new attribute to spans without the specified key.
+- update: Updates an attribute of spans with the specified key.
+- upsert: Performs insert or update. Inserts an attribute for spans without
+  the specified key and updates an attribute for spans with the specified key.
+- delete: Deletes an attribute from a span.
+
+**INSERT is the default action.**
+
+
+For the actions `insert`, `update` and `upsert`, a `key` must be specified,
+ one of `value` or `from_attribute` and `action`.
```

Comment:
A possible expansion for the future `from_environment` and others seem likely to happen.